wagtail/snippets/action_menu.py

```diff
@@ -217,7 +218,14 @@ class DeleteMenuItem(ActionMenuItem):
     name = "action-delete"
     label = _("Delete")
     icon_name = "bin"
-    classname = "action-secondary"
+
+    def __init__(self, order=None):
+        super().__init__(order)
+        warn(
+            "DeleteMenuItem is deprecated. "
+            "The delete option is now provided via EditView.get_header_more_buttons().",
```

Comment:
I was wondering whether this should be documented, but turns out `DeleteMenuItem` isn’t documented either, so I think we’re good :)